rocketpy/sensitivity/sensivity_model.py

```diff
@@ -0,0 +1,515 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import norm
+from ..tools import check_requirement_version, import_optional_dependency
+
+
+class SensitivityModel:
+    """Performs a 'local variance based first-order
+    sensitivity analysis' considering independent input parameters.
+
+    The core reference for global variance based sensitivity analysis is
+    [1]. Our method implements a local version that only considers first
+    order terms, which correspond to linear terms. Albeit the flight
+    function is nonlinear, the linear hypothesis might be adequate when
+    performing *local* sensitivity analysis.
+
+    The model is fit using separate multiple linear regression for each
+    target variable passed and using the parameters as covariates.
+
+    References
+    ----------
+    [1] Sobol, Ilya M. "Global sensitivity indices for nonlinear mathematical models and their Monte Carlo estimates." Mathematics and computers in simulation 55.1-3 (2001): 271-280.
```

Comment:
Line too long... You can use the `\` sign to continue the text in the next line whithout breaking it. Like this:

"something here \
    continues here, do you see?"